### PR TITLE
emphasize deletion of all signatures if using invalid identity

### PR DESCRIPTION
**Pull request by wozniakjan**
This is to supplement https://github.com/openshift/origin/pull/19976, @bmcelvee  in case you already created the PR we talked about, feel free to close this one. I was trying to find yours but couldn't and I had some spare time on Friday for this anyway :)

ptal @bmcelvee, @bparees

### CODE REVIEW

File: admin_guide/image_signatures.adoc [hunk 1]
```diff
@@ -124,9 +124,15 @@ $ oc adm policy add-cluster-role-to-user system:image-auditor <user_name>
 endif::[]
 ====
 
+[IMPORTANT]
+====
 Using the `--save` flag on already verified image together with invalid GPG key
-or invalid expected identity causes the saved verification status to be removed,
-and the image will become unverified.
+or invalid expected identity causes the saved verification status with all
```
Comment (bparees) [inline admin_guide/image_signatures.adoc]: "with all" -> "and all"?

File: admin_guide/image_signatures.adoc [hunk 2]
```diff
@@ -124,9 +124,15 @@ $ oc adm policy add-cluster-role-to-user system:image-auditor <user_name>
...
+previous signatures to be removed, and the image will become unverified.
```
Comment (bparees) [inline admin_guide/image_signatures.adoc]: s/previous// ?

Comment (wozniakjan) [reply admin_guide/image_signatures.adoc]: couldn't come up with any better adjective so I just removed the 'previous'

Comment (bparees) [reply admin_guide/image_signatures.adoc]: sorry if it wasn't clear, that's exactly what i was suggesting you do :)

**Comment by bparees** [comment]
lgtm
@openshift/team-documentation ptal/merge

**Comment by bmcelvee** [comment]
Thanks, @wozniakjan! LGTM. The bug attached to this is set for OCP 3.6, so is this applicable for 3.6+?

No QE required.

**Comment by bparees** [comment]
yes, 3.6+

**Pull request merged**